src/main/scala/io/digitallibrary/bookapi/service/ExportService.scala

```diff
@@ -186,6 +189,9 @@ trait ExportService {
       generator.writeRawValue(s"$baseUrl/${languageTag.toString}/books/details/${book.id}")
       generator.writeRawValue("")
       generator.writeRawValue("")
+      generator.writeRawValue(book.readingLevel.getOrElse(""))
+      generator.writeString(book.coverImage.map(i => i.url).getOrElse(""))

```

Comment:
cloudinary-urler har , som en del av urlen og det passer dårlig i en kommaseparert fil...

Reply:
Har ikke satt meg nok inn i det, men når vi testet med urlene kan "," erstattes med "%2C"

@Erlendgje vet du hvor urlene kommer fra? :)

Reply:
Sikkert lurere å urlencode ja. I ndla har vi brukt lemonlabs scala-uri. Kan teste ut det.

Reply:
Her linje 91 https://github.com/GlobalDigitalLibraryio/image-api/blob/master/src/main/scala/no/ndla/imageapi/ImageApiProperties.scala , hilsen @Erlendgje

Reply:
Kunne faktisk bytta ut , i den urlen med %2C i stedet for å bruke eksternt bibliotek.

Reply:
Men då må eg også oppdatere image-api...

Reply:
Fiksa det ved å parse url og så legge på en implicit urlconfig.